index.js

```diff
@@ -113,6 +114,10 @@ module.exports = {
   },
 
   init: function (app) {
+    // Determine if the ember version is >= 2.12
+    const checker = new VersionChecker(this)
```

Comment:
We can do it in one or the other. I'm fine with doing this in `treeForAddon`. @juwara0 any opinion?